base/MAPL_NetCDF.F90

```diff
@@ -18,282 +14,247 @@ module MAPL_NetCDF
 
    implicit none
 
-   public :: convert_NetCDF_DateTime_to_ESMF
-   public :: convert_ESMF_to_NetCDF_DateTime
```

Comment:
I really need these two subroutines.  My code is now depending on the `convert_NetCDF_DateTime_to_ESMF` subroutine which provides a complete set of information that is very helpful.    Please keep these two subroutines.